spmd/compiler/utils.py

```diff
@@ -0,0 +1,18 @@
+import logging
```

Comment:
minor nit: having generic file names like 'utils.py' is always scary to me b/c of the risk of confusion when every dir has it's own utils.py.  
In the future, could we rename to log_utils.py or similar to keep things distinct?  I'll have a graph_utils.py coming for example. 
We can update this in the future though, not worth changing for now.